lib/press/neueburcherbeitung.svg

```diff
@@ -0,0 +1,6 @@
+<svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 893.869 118.841">
+  <title>Neue Burcher Beitung Logo</title>
```

Comment:
Can you add a `<desc>` tag?  the other ones have them as well if you need an example